docs/reference/logging/overview.md

```diff
@@ -172,3 +173,31 @@ The Amazon CloudWatch Logs logging driver supports the following options:
 
 
 For detailed information on working with this logging driver, see [the awslogs logging driver](awslogs.md) reference documentation.
+
+## Specify splunk options
```

Comment:
@outcoldman if you can move this text to a separate document ([see the page for journald as example](https://github.com/docker/docker/blob/master/docs/reference/logging/journald.md)), and only keep an introduction here (similar to [the journald driver](https://github.com/docker/docker/blob/master/docs/reference/logging/overview.md#journald-options)